### PR TITLE
Document AvFILLp

### PR DESCRIPTION
**Pull request by khwilliamson**

### CODE REVIEW

File: av.h [hunk 1]
```diff
@@ -50,7 +50,10 @@ Same as C<av_top_index()> or C<av_tindex()>.
 =for apidoc av_tindex
 Same as C<av_top_index()>.
 
-=for apidoc m|int|AvFILLp|AV* av
+=for apidoc Cm|int|AvFILLp|AV* av
+
+This returns the topmost index of the array C<a>, or -1 if the array is empty.
```
Comment (tonycoz) [inline av.h]: To me the "topmost index" of an array is zero.

**Comment by khwilliamson** [comment]
On 9/8/20 5:55 PM, Tony Cook wrote:
> *@tonycoz* commented on this pull request.
> 
> ------------------------------------------------------------------------
> 
> In av.h <https://github.com/Perl/perl5/pull/18113#discussion_r485257726>:
> 
>> @@ -50,7 +50,10 @@ Same as C<av_top_index()> or C<av_tindex()>.
>   =for apidoc av_tindex
>   Same as C<av_top_index()>.
>   
> -=for apidoc m|int|AvFILLp|AV* av
> +=for apidoc Cm|int|AvFILLp|AV* av
> +
> +This returns the topmost index of the array C<a>, or -1 if the array is empty.
> 
> To me the "topmost index" of an array is zero.
> 
I wonder if this hemipherist bias on my/your part.  ;)
A colleague of mine had macarthur's corrective map of the world
  on his wall.

The term 'topmost' conforms to the function already in core 
'av_top_index', which I gather is also confusing to you.

I'm happy to accept a better alternative phrasing,  but I never came up 
with something as concise.


"the index of the array element currently furthest away from 0" for example

**Comment by tonycoz** [comment]
> I wonder if this hemipherist bias on my/your part. ;) A colleague of mine had macarthur's corrective map of the world on his wall.

Hah :)

> The term 'topmost' conforms to the function already in core 'av_top_index', which I gather is also confusing to you. I'm happy to accept a better alternative phrasing, but I never came up with something as concise. "the index of the array element currently furthest away from 0" for example

"last"?

Yes, I'm not fond of av_top_index() either, but please don't define yet another name for it.

left/right obviously don't work either.

**Comment by khwilliamson** [comment]
@tonycoz How about this revised wording?

**Comment by tonycoz** [comment]
> @tonycoz How about this revised wording?

That's better.

**Pull request closed without merging**